package.json

```diff
@@ -11,16 +11,14 @@
     "build:chrome": "PLATFORM=chrome node scripts/build.js",
     "build:firefox": "PLATFORM=firefox node scripts/build.js",
     "build": "yarn build:chrome && yarn build:firefox",
-    "fix": "yarn fix:js && yarn fix:other",
-    "fix:other": "yarn prettier",
-    "fix:js": "yarn test:js --fix",
-    "prettier": "prettier . --write",
-    "build:chrome:dev": "PLATFORM=chrome node scripts/start.js",
-    "build:firefox:dev": "PLATFORM=firefox node scripts/start.js",
+    "dev:chrome": "PLATFORM=chrome node scripts/start.js",
+    "dev:firefox": "PLATFORM=firefox node scripts/start.js",
     "launch:firefox": "web-ext run",
-    "test": "yarn test:other && yarn test:js && jest --passWithNoTests",
-    "test:other": "prettier . --check",
-    "test:js": "eslint --ignore-path .gitignore --ignore-path .prettierignore \"**/*.js\""
+    "lint": "eslint --ignore-path .gitignore \"**/*.js\"",
+    "format": "prettier . --write",
+    "format:check": "prettier . --check",
+    "test": "jest --passWithNoTests",
+    "ci": "yarn format:check && yarn test"
```

Comment:
I don't think we need this `ci` script, please refer to my comment about enabling ESLint in `lint-staged`.